samples/SkiaNativeSample/src/macosX64Main/kotlin/org.jetbrains.skiko.sample.native/App.kt

```diff
@@ -50,14 +51,17 @@ fun displayScene(renderer: Renderer, nanoTime: Long) {
     val canvas = renderer.canvas!!
 
     val paint = Paint()
-    paint.setColor(SK_ColorGREEN)
+    //paint.setColor(Color.GREEN)
+    paint.setColor(0xFF00FF00.toInt())
 
-    canvas.clear(SK_ColorRED);
+    // canvas.clear(Color.RED);
+    canvas.clear(0x00FF00FF.toInt());
 
     canvas.save();
-    canvas.translate(128.0f, 128.0f)
-    canvas.rotate(nanoTime.toFloat() / 1e7f)
-    val rect = SkRect.MakeXYWH(-90.5f, -90.5f, 181.0f, 181.0f)
+    // TODO: disabled to ramp up the new native skiko.
```

Comment:
What's wrong here, why we shall do that?

Reply:
Those eventually pass a Matrix, which goes to the native bindings as `jfloatArray` or something. Arrays are not working yet.